Sources/Clappr_iOS/Classes/Animation/SeekBubble/SeekBubble.swift

```diff
@@ -0,0 +1,224 @@
+class SeekBubble: UIView {
+    private var label = UILabel()
+    private var images: [UIImageView] = []
+    private var text: String!
+    
+    private var parentView: UIView!
+    
+    var bubbleHeight = NSLayoutConstraint()
+    var bubbleWidth = NSLayoutConstraint()
+    var bubbleSide: SeekBubbleSide!
+    
+    func setup(within parentView: UIView, bubbleSide: SeekBubbleSide, text: String = "10 segundos", numberOfIndicators: Int = 3) {
+        self.parentView = parentView
+        self.bubbleSide = bubbleSide
+        self.text = text
+        parentView.clipsToBounds = true
+        
+        setupBubble(within: parentView, heightConstraint: &bubbleHeight, widthConstraint: &bubbleWidth, position: bubbleSide.position())
```

Comment:
If the `bubbleHeight` and `bubbleWidth` are properties of this class, why to not just access it using `self`?